c/driver/postgresql/postgresql_test.cc

```diff
@@ -143,10 +143,17 @@ TEST_F(PostgresConnectionTest, GetInfoMetadata) {
           EXPECT_EQ("PostgreSQL", std::string(val.data, val.size_bytes));
           break;
         }
-        case ADBC_INFO_VENDOR_VERSION:
-          // UTF8
-          ASSERT_EQ(uint8_t(0),
-                    reader.array_view->children[1]->buffer_views[0].data.as_uint8[row]);
+        case ADBC_INFO_VENDOR_VERSION: {
+          ArrowStringView val = ArrowArrayViewGetStringUnsafe(str_child, 3);
+#ifdef __WIN32
```

Comment:
This is unfortunate but I don't think there is a cross platform way for this to work https://github.com/google/googletest/issues/3084